src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -171,4 +192,46 @@ public void allowOverrideRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() thr
 
     assertThat(response.getJson().getString("action"), is("renewedThroughOverride"));
   }
+
+  @Test
+  public void forbidRenewalOverrideWhenFirstRequestIsNotRecall()
+    throws MalformedURLException,
+    InterruptedException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    DateTime loanDueDate = new DateTime(
```

Comment:
This is interesting formatting. Can you help me understand why this date has been formatted this way?

I'd like other folks thoughts on whether this improves readability @mreno-EBSCO @mtraneis @kostyantyn-kh ?

Reply:
It was just formatted in the same way as in other tests in this class.
Do you think it'd be better to change the formatting in all places?

Reply:
This was the first time I noticed the formatting. It is unusual, maybe the original author can explain the reasoning behind it?

Reply:
Seems like it's done to fit 80 character line.
![image](https://user-images.githubusercontent.com/42538529/59765891-bbd4f400-92a7-11e9-86bc-116bb1895b84.png)
![image](https://user-images.githubusercontent.com/42538529/59765959-dc04b300-92a7-11e9-8542-9ba8909fc229.png)

Reply:
If the length exceeds 80 characters, personally I'd split it at between the date and the time, if that makes it fit better.

(I'm looking forward to Java 11, so can use implicit variables to shorten some of this)